Tech@HubAPI/Tech@HubAPI/Controllers/UserController.cs

```diff
@@ -145,8 +145,12 @@ public ActionResult<User> SignUp([FromBody] SignUpForm form)
 
             DateTime.TryParse(form.BirthDate, out DateTime birthDate);
 
+            var rand = new Random();
+            var files = Directory.GetFiles("Profile_Pictures_JPG/", "*.jpg");
```

Comment:
Lets also add a check here saying if the directory doesn't exist or the length of the files array is 0 (ie, there are no files), just set the `randomPfpPicture` variable to an empty string. That way we won't have to worry about mocking this folder in the unit tests